### PR TITLE
Added footer + the first icons 

### PR DESCRIPTION
**Pull request by DErdmann1**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -8,14 +8,71 @@ header {
...
+    position: fixed;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    background-color: lightgrey;
+    height: 60px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    justify-content: space-between;
```
Comment (vaupunkt) [inline styles.css]: i think "space around" works better here

File: styles.css [hunk 2]
```diff
@@ -8,14 +8,71 @@ header {
...
 
+.footer {
+    position: fixed;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    background-color: lightgrey;
+    height: 60px;
+    display: flex;
+    justify-content: center;
```
Comment (vaupunkt) [inline styles.css]: you have justify content 2 times

File: styles.css [hunk 3]
```diff
@@ -8,14 +8,71 @@ header {
...
+.questions {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: 40px;
+    border: 2px solid black;
+    max-width: 700px;
+    padding: 50px;
+    margin-left: 20px;
+    margin-right: 20px;
```
Comment (vaupunkt) [inline styles.css]: if you change your resolution the questions aren't centered. maybe you could go with "margin: auto"?

File: index.html [hunk 4]
```diff
@@ -6,8 +6,86 @@
...
+              <a>flexbox</a>
+            </div>
+            <div class="categorys">
+              <a>css</a>
+            </div>
+          </section>
+    </section>
+    <br>
+    <br>
+    <br>
```
Comment (vaupunkt) [inline index.html]: i think this is easer with padding, than with linebreaks

File: styles.css (hunk 3 above)
Comment (DErdmann1) [reply styles.css]: Great idea! **Thanks** :)

File: index.html [hunk 5]
```diff
@@ -6,8 +6,86 @@
...
     <link rel="stylesheet" type="text/css" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
 </head>
 <header>
     <h1>Quiz App</h1>
 </header>
+<main>
+<body>
+    <section class="questions">
+        <p><strong>In Zukunft werden hier tolle Fragen stehen, richtig?</strong></p>
```
Comment (vaupunkt) [inline index.html]: you could change this in css, which would make you html look cleaner and you wouldn't have to add it for every question

**Review by vaupunkt — APPROVED** [review]
some smaller comments, but overall i think you are on a good way! 😃

**Pull request merged**